### PR TITLE
Desktop-version changes

### PR DESCRIPTION
**Pull request by Marlyn-Mayienga**
# # Desktop-version changes
In this module, the following changes were made:
- made changes to the background images in the portfolio
- modified the desktop version to fit the design
- changed the styling to align the icons and text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,95 +25,333 @@
...
-              <li>GitHub</li>
-              <li>CodePen</li>
-              <li>GitLab</li>
-              <li>Terminal</li>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li>
+                <a href="#"><i class="far fa-envelope"></i></a>
+              </li>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly remove the underline from the nav links to match the Figma design.
- [ ] Kindly change the nav links color to white to match the Figma design.

Figma design
image


Your design
image

File: index.html [hunk 2]
```diff
@@ -25,95 +25,333 @@
...
+                  <a href="marlymayienga@gmail.com"
+                    ><i class="fas fa-envelope"></i
+                  ></a>
+                </li>
+              </ul>
+            </div>
+            <div>
+              <p>&darr; Work</p>
+            </div>
+          </section>
```
Comment (keroloussamy) [inline index.html]: - [ ] Kindly change the color for the paragraph and the technologies to `#B3BAC5;` to match the Figma design.
- [ ] Kindy make the two buttons `Work` a little bit away from the accounts icons.

Figma design
image


Your design
image

File: style.css [hunk 3]
```diff
@@ -305,9 +329,15 @@ h3 {
...
+/* .content {
+  background: url("/circle_header.svg") no-repeat;
+  background-position: 150% 150%;
+  background-size: 200px;
+} */
+
+/* .contact-title.desktop {
   display: block;
-}
+} */
```
Comment (keroloussamy) [inline style.css]: - [ ] Kindly remove the commented code, you can find it on the source tree always.

File: index.html [hunk 4]
```diff
@@ -25,95 +25,333 @@
...
+
+              <div class="work-item-content">
+                <img class="work-img" src="./project.jpg" alt="project image" />
+                <div class="work-content">
+                  <h4 class="work-item-title">Multi-Post Stories</h4>
+                  <p class="project-section">
+                    A daily selection of privately personalized reads; no
+                    accounts or sign-ups required. has been the industry's
+                    standard dummy text ever since the 1500s, when an unknown
+                    printer took a standard dummy text.
```
Comment (keroloussamy) [inline index.html:171]: - [ ] Kindly change the color for the paragraph to `#B3BAC5` to match the Figma design.

File: index.html [hunk 5]
```diff
@@ -25,95 +25,333 @@
...
+              rows="10"
+              cols="50"
+              required
+              minlength="1"
+              maxlength="500"
+            ></textarea>
+
+            <button type="submit" class="send-button">Send</button>
+          </form>
+        </section>
```
Comment (keroloussamy) [inline index.html:338]: - [ ] Kindly your mobile version still looks very good, just one thing you have to work on is adding some padding from left and right on the contact section to match the Figma design.

Figma design
image

Your design
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Marlyn-Mayienga, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Marlyn-Mayienga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**